lmfit/minimizer.py

```diff
@@ -270,7 +278,7 @@ def values(self):
 
         return dict([(name, p.value) for name, p in self.result.params.items()])
 
-    def __residual(self, fvars):
+    def __residual(self, fvars, apply_scaling=True):
         """
         Residual function used for least-squares fit.
         With the new, candidate values of fvars (the fitting variables), this
```

Comment:
The docstring needs to be updated (it needed overhauling anyway):

    Parameters
    ----------------
    fvars : np.ndarray
        <what are fvars>
    apply_scaling : bool, optional
         <what is apply_scaling>